### PR TITLE
Added Chambery tests

### PR DESCRIPTION
**Pull request by avulfson17**
Related to discussion in pelias/api#600

### CODE REVIEW

**Comment by trescube** [comment]
:shipit:

**Comment by orangejulius** [comment]
I fixed your description to remove an extra slash, as the ticket wasn't linking correctly :)

File: test_cases/encoding.json [hunk 1]
```diff
@@ -44,6 +44,48 @@
           }
         ]
       }
+    },
+    {
+      "id": 3,
+      "status": "pass",
+      "endpoint": "search",
+      "description": "the Chambéry accent issue, with the accented e represented as 1 character",
```
Comment (orangejulius) [inline test_cases/encoding.json]: Adding a link to the original issue (https://github.com/pelias/api/issues/600) would be great here and in the next test

**Comment by dianashk** [comment]
LGTM

File: test_cases/encoding.json (hunk 1 above)
Comment (orangejulius) [inline test_cases/encoding.json]: Now that you get github emails you will see this message referring to my previous comment which says "Adding a link to the original issue (pelias/api#600) would be great here and in the next test"

**Comment by orangejulius** [comment]
LGTM too

**Comment by orangejulius** [comment]
I rebased this PR against the latest master so the Travis tests would run. it _still_ LGTM :)

**Pull request merged**